azure-pipelines.yml

```diff
@@ -132,10 +139,27 @@ stages:
       displayName: 'Install matplotlib before refguide run'
     - script: python runtests.py -g --refguide-check
       displayName: 'Run Refuide Check'
+      condition: eq(variables['USE_OPENBLAS'], '1')
     - script: python runtests.py -n --mode=full -- -rsx --junitxml=junit/test-results.xml
       displayName: 'Run Full NumPy Test Suite'
+      condition: eq(variables['USE_OPENBLAS'], '1')
     - bash: python tools/openblas_support.py --check_version
       displayName: 'Verify OpenBLAS version'
+      condition: eq(variables['USE_OPENBLAS'], '1')
+    # import doesn't work when in numpy src directory , so do a pip dev install of build lib to test
+    - script: |
+        #!/bin/bash -v
+        set +e
+        python -m pip install -e .
```

Comment:
According to [the logs](https://dev.azure.com/numpy/numpy/_build/results?buildId=8555&view=logs&j=5e4e1a45-8952-53a3-057c-01290c9cd1c2&t=9c44b4dc-5221-509c-54af-0b1de3567cbb&l=21), this line re-builds numpy. Can you remove it? Note the `python runtest.py` on line 135/143 has a `-n` to skip building, since numpy is built and installed on line 121/128